packages/ra-ui-materialui/src/list/datagrid/expandAllButton.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import { Expand } from './Datagrid.stories';
+
+describe('ExpandAllButton', () => {
+    it('should expand all rows at once', async () => {
+        const rendered = render(<Expand />);
+        const button = rendered.container.querySelector(
```

Comment:
instead of `container`, we tend to use aria labels or data-testid to locate elements without text - just like a screen reader would do. See https://testing-library.com/docs/guiding-principles